### ISSUE TITLE
TypeError: Cannot read property 'power' of undefined

### ISSUE THREAD

**Issue by xuqisheng** [seikan/homebridge-mi-air-purifier#3]
crash when start.
```{r, engine='bash', count_lines}
/usr/lib/node_modules/homebridge-mi-air-purifier/index.js:97
                callback(null, this.device.power);
                                          ^

TypeError: Cannot read property 'power' of undefined
    at Object.getOn (/usr/lib/node_modules/homebridge-mi-air-purifier/index.js:97:29)
    at emitThree (events.js:116:13)
    at emit (events.js:194:7)
    at Characteristic.getValue (/usr/lib/node_modules/hap-nodejs/lib/Characteristic.js:129:10)
    at Bridge.<anonymous> (/usr/lib/node_modules/hap-nodejs/lib/Accessory.js:650:20)
    at Array.forEach (native)
    at Bridge.Accessory._handleGetCharacteristics (/usr/lib/node_modules/hap-nodejs/lib/Accessory.js:614:8)
    at emitMany (events.js:127:13)
    at HAPServer.emit (events.js:201:7)
    at HAPServer._handleCharacteristics (/usr/lib/node_modules/hap-nodejs/lib/HAPServer.js:926:10)
```

**Comment by seikan** [comment]
This error has been resolved in version [1.1.0](https://github.com/seikan/homebridge-mi-air-purifier/releases/tag/1.1.0).

**Comment by xuqisheng** [comment]
Thank you very much👏
